5_ReduxBasic/src/App.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './App.css';
+import {credit1AC, credit2AC, debitAC, setBalanceAC, updateBalanceAC} from "./Redux/Reducer";
+import {useDispatch, useSelector} from "react-redux";
+import {reducer} from "./Redux/Reducer";
+import {} from "./Redux/Store"
+
+
+// -----------------------------STORE with redux dev tool------------------
+import { createStore, compose } from "redux";
+
+  declare global {
+  interface Window {
+    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
+  }
+}
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+export const store = createStore(reducer, undefined, composeEnhancers());
+//@ts-ignore
+window.store = store;
+
+// ---------------------------------------------------------------------
+
+
+function App() {
+
+    const balance=useSelector<any>(state=>state.number)
```

Comment:
why any? for one field it would be great to use types as well due to having it below